apps/domain/src/main/routes/general/routes.py

```diff
@@ -39,3 +39,9 @@ def root_route():
     else:
         node.recv_eventual_msg_without_reply(msg=obj_msg)
     return ""
+
+
+@root_route.route("/", methods=["GET"])
+def get_setup():
+    response = {"message": "Domain API deployment successful"}
+    return Response(json.dumps(response), status=200, mimetype="application/json")
```

Comment:
What is the use of this route inside general? It should be in the setup route instead, right?

Reply:
It can be anywhere, it's just for testing purpose, and would be removed at later point of time.

Reply:
for testing purposes, we should use the infra routes instead! I'm gonna remove this